app/README.md

```diff
@@ -27,7 +27,42 @@ Build and package the app:
 2. Get the generated package from: `dist/radicle-upstream-0.0.1.dmg`
 
 
-#### Buildkite
+#### Running tests locally
+
+Before running tests locally you'll need to set up a test fixture repository
+via: `cd ../../ && git clone https://github.com/radicle-dev/git-platinum`.
```

Comment:
Wouldn't it be better to have the repo be set up inside of radicle-upstream? Otherwise you have to make assumptions for any state outside of the current repo.

Additionally a simple clone will not pin it to the right version that is known to work for the current test suite. A way to pin is for example git submodules.

Reply:
I wanted to have it as a git submodule initially, but then I realised that we depend on the state of the test repository as a whole.

We don't need it to be pinned to a specific version, because we're querying all kinds of data that don't depend on the current checkout, for example:
  - which files exist at a specific revision
  - all tags/branches

Reply:
@garbados that's exactly what we need, it should be outside of this repository, as we don't need it to be "mounted" as a git-submodule this repository. We need the layout to be like this:
```
workdir/
.
├── git-platinum
└── radicle-upstream
```

Reply:
I think that's the point, that it makes an assumption about the state outside of the repository directory boundaries.

Reply:
I updated the README to make it more clear.

Reply:
Y'all were right, I was wrong. I got it working as a submodule, much cleaner now.

Reply:
:heart: